### DISCUSSION TITLE
[...path].tsx

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#719] [Q&A]
<Head>
        <title>{siteConfig.title}</title>
      </Head>

we want change this tag {siteConfig.title} to read current file or folder name so how we can change this 
@myl7

**Comment by myl7** [comment]
Here the code `<title>{siteConfig.title}</title>` will be run on client side (in user browsers), so if you need to get the title from filesystem:

- If you can pack those files into the repo to provide them in Vercel runtime, you may use Next.js [`getServerSideProps`](https://nextjs.org/docs/basic-features/data-fetching/get-server-side-props) to fetch it.
- If you can give the title via an env, Next.js [`NEXT_PUBLIC_` prefixed env](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser) can help. Personally speaking I would recommend this solution.

**Reply by hirockstar** [reply]
can you help me to do that
